### PR TITLE
dvclive: Use admonitions

### PR DESCRIPTION
**Pull request by daavoo**

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]

File: content/docs/dvclive/get-started.md [hunk 1]
```diff
@@ -3,8 +3,12 @@
 DVCLive is a Python library for logging machine learning metrics and other
 metadata in simple file formats, which is fully compatible with DVC.
 
-> If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks),
-> you can jump directly to it's corresponding page.
+<admon type="tip">
+
+If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
+can jump directly to it's corresponding page.
```
Comment (rogermparent) [inline content/docs/dvclive/get-started.md:9]: While not added in this PR, I noticed this should be "its corresponding page". Weird little English exception to pluralizing.

File: content/docs/dvclive/api-reference/live/index.md [hunk 2]
```diff
@@ -25,9 +25,13 @@ live = Live()
 A `Live()` instance is required in order to log machine learning metrics and
 other metadata.
 
-⚠️ `Live()` will remove all existing DVCLive related files under `path` unless
+<admon type="warn">
+
+`Live()` will remove all existing DVCLive related files under `path` unless
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/live/index.md:30]: Does Prettier complain if you don't leave a blank line?

File: content/docs/dvclive/dvclive-with-dvc.md [hunk 3]
```diff
@@ -116,7 +116,11 @@ plot for the logged data automatically updated during the model training!
 
 ![](/img/dvclive-html.gif)
 
-> If you don't update the step number, the HTML report won't be generated.
+<admon type="info">
+ 
+If you don't update the step number, the HTML report won't be generated.
+
+</admon>
```
Comment (jorgeorpinel) [inline content/docs/dvclive/dvclive-with-dvc.md:123]: Did you mean that text to be highlighted though? Block quotes can still be used for text we want to make less relevant (something you can skip, but without completely hiding it in a `<details>`).

Comment (jorgeorpinel) [reply content/docs/dvclive/dvclive-with-dvc.md:123]: Q applies to every other non-emoji note turned into `<admon type="info">`.

File: content/docs/dvclive/api-reference/live/index.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/dvclive/api-reference/live/index.md:30]: I guess the code is easier to read and we do this with `<details>` as well. Up to you

File: content/docs/dvclive/dvclive-with-dvc.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/dvclive/dvclive-with-dvc.md:123]: ☝🏼 Up to you

Comment (daavoo) [reply content/docs/dvclive/dvclive-with-dvc.md:123]: Yes, I wanted it to be highlighted although probably not taking as much space (#3381 )

**Pull request merged**